scripts/src/utils/utils.ts

```diff
@@ -69,14 +71,21 @@ export enum IdPrefix {
 const ID_LENGTH = 20;
 const ID_CHARS = "1234567890abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ";
 
+/**
```

Comment:
yes, this is good to have doc comments, but what format have you chosen here?

Reply:
Doxygen, I like it much

Reply:
oh, well, i prefer working with [tsdoc](https://github.com/Microsoft/tsdoc) as it makes more sense because it's for typescript and works like javadoc (it infers the types)